packages/webapp/pages/onboarding.tsx

```diff
@@ -176,16 +178,35 @@ export function OnboardPage(): ReactElement {
   const [isPlusCheckout, setIsPlusCheckout] = useState(false);
   const hasSelectTopics = !!feedSettings?.includeTags?.length;
 
+  const isOnboardingReady = isAuthReady && (isActionsFetched || !user);
+  const { value: isReorderExperiment } = useConditionalFeature({
+    feature: featureOnboardingReorder,
+    shouldEvaluate: isOnboardingReady && !user,
```

Comment:
```suggestion
    shouldEvaluate: isOnboardingReady && !user && showOnboardingPage,
```

Reply:
Could not put the whole `showOnboardingPage ` because this experiment need to be evaluated not depending on `isAuthenticating`. I've added ` && isIntro && !shouldVerify` still ✔️